keps/sig-node/1769-memory-manager/README.md

```diff
@@ -435,21 +501,19 @@ Memory pinning will be validated for Topology Manager `single-numa-node` and `re
 
 #### GA (stable)
 
-- User feedback
-- TBD
-
-<!--
-#### Removing a deprecated flag
+- N examples of real-world usage
+- N installs
+- More rigorous forms of testing—e.g., downgrade tests and scalability tests
+- Allowing time for feedback
 
-- Announce deprecation and support policy of the existing flag
-- Two versions passed since introducing the functionality which deprecates the flag (to address version skew)
-- Address feedback on usage/changed behavior, provided on GitHub issues
-- Deprecate the flag
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports,
+in back-to-back releases.
 
-**For non-optional features moving to GA, the graduation criteria must include [conformance tests].**
+**For non-optional features moving to GA, the graduation criteria must include
+[conformance tests].**
```

Comment:
we need to state whether we will have any conformance tests. I don't think we can have any so I would just say that this is not applicable

Reply:
Why can't we have conformance tests?

Reply:
conformance tests validate an API endpoints. There is no APIs related to memory manager to test

Reply:
agreed and added to the KEP text.